uwblueprint/website-bp
# Intw24 review dashboard data

## Notion ticket link

[Review Dashboard Data](https://www.notion.so/uwblueprintexecs/Review-Dashboard-Data-1666c0ae8e3f4fbaace398fc7c9131fe?pvs=4)



## Implementation description
* Edit and Save button working to edit and save scores
* Front end displays scores stored in database



## Steps to test
1. Go to /admin
2. Project Developer
3. Review Dashboard
4. Under Kushal Goel's application, there should be 2 reviews for which the score corresponds to the data within the database
5. To test edit and save functionalities, edit the scores by clicking the edit button. Change the scores to whatever you like and press save. The scores should be updated in the db and front end.



## What should reviewers focus on?
* 


## Checklist
